template/shell.nix

```diff
@@ -0,0 +1,9 @@
+let
+  scheme =
+    if builtins.pathExists ./.git
+    then "git+file"
+    else "path";
+in
+  (builtins.getFlake "${scheme}://${toString ./.}")
```

Comment:
I don't know if that would work. Does `builtins.getFlake` work if flakes are disabled?

Reply:
That's a really good question.  I'll have to check and get back to you on that

Reply:
Works with vanilla `nix-shell` as well now